view/frontend/templates/js/checkout_funnel.js.phtml

```diff
@@ -1,7 +1,9 @@
 <?php
 if (!$block->shouldTrackCheckoutFunnel()) return;
+
+$objectManager =  \Magento\Framework\App\ObjectManager::getInstance();
```

Comment:
We shouldn’t use the Object Manager to initialize an object. The best practice here is to inject the track-checkout.js using RequireJs
Something like this:
> <script type="text/x-magento-init">
>     {
>         "*" : {
>             "Bolt_Boltpay/js/track-checkout" : {}
>         }
>     }
> </script>

Reply:
nice I had hard time figuring out this; thank you